tests/test_pillow_legacy.py

```diff
@@ -381,15 +391,16 @@ def test_inside_zipfile():
         imageio.imread(fname + "/" + name)
 
 
-def test_bmp():
-    need_internet()
-    fname = get_remote_file("images/scribble_P_RGB.bmp", test_dir)
+@pytest.mark.needs_internet
+def test_bmp(tmp_path):
+    fname = get_remote_file("images/scribble_P_RGB.bmp", tmp_path)
```

Comment:
We got another one! I think at this point 50 of my comments must be on the amazing `test_images` fixture :D